### DISCUSSION TITLE
multiwallet

### DISCUSSION THREAD

**Discussion by karapy** [dogecoin/dogecoin#2939] [Q&A]
Hello, 

I am trying to have 2 wallet but when I start dogecoind by -wallet I can not access 
http://127.0.0.1:22555/wallet/wallet.dat

how can I have multi wallet on dogecoin core wallet 

Dogecoin Core RPC client version v1.14.5.0-31afd1331
I am using ubuntu Server
I am using dogecoin core to generate addresses and get notification of transactions

**Comment by patricklodder** [comment]
Hi! 1.14 only supports one active wallet. We'll get the multi-wallet functionality with the 1.21 release.

**Reply by karapy** [reply]
> Hi! 1.14 only supports one active wallet. We'll get the multi-wallet functionality with the 1.21 release.

What is the solution till then?

**Reply by patricklodder** [reply]
There is none right now short of running more than one node, sorry.
